addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/dashboard/Dashboard.js

```diff
@@ -115,18 +126,17 @@ class Dashboard extends React.Component {
   }
 
   async getCosts() {
-    const { envNameToCostInfo, envNameToIndex } = await this.getAccumulatedEnvCost();
+    const { envIdToCostInfo, envIdToEnvMetadata, duplicateEnvNames } = await this.getAccumulatedEnvCost();
 
     const projNameToUserTotalCost = {};
-    Object.keys(envNameToCostInfo).forEach(envName => {
-      const projName = envNameToIndex[envName];
+    Object.keys(envIdToCostInfo).forEach(envId => {
+      const projName = envIdToEnvMetadata[envId].index;
```

Comment:
ProjectName is stored under key index?

Reply:
I think it is a bad variable name. The entire file is using project name at all places, I can rename the entire file (around 48 usages)